### PR TITLE
StreamListRow Component 

### PR DESCRIPTION
**Pull request by asharma8602**
Please suggest improvements .
Also I had updated the packages to those which were added last year as i was facing typescript version issue.
Thanks

### CODE REVIEW

File: ui/screens/StreamListRow.tsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+export default function App() {
+  return (
+    <ScrollView>
+      <Header
+        leftComponent={{ icon: 'home', color: '#fff'  }}
+        centerComponent={{ text: 'Stream List', style: { 
+          textAlign: 'center',
+          fontSize: 25,
+          color: '#fff',
+          fontFamily: 'Baskerville Old Face'
```
Comment (mridul911) [inline ui/screens/StreamListRow.tsx:57]: Remove this font it crashes for android.

File: ui/screens/StreamListRow.tsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    pk: 2,
+    title: 'Craze of Club House',
+    description: 'Recently People are shifting to clubhouse',
+    followed_by: 400,
+  },
+
+];
+
+
+export default function App() {
```
Comment (mridul911) [inline ui/screens/StreamListRow.tsx:48]: use arrow functions like const StreamListRow = () => {}  and then export it so it can be used.

**Review by mridul911 — CHANGES_REQUESTED** [review]
We will be getting 'is_subscribed' also we can show that in right as a chip or button red for not subscribed and blue for other one. You can also add some background style.

**Comment by 7shivamx** [comment]
I'm taking up this issue

**Pull request closed without merging**